plugins/org.eclipse.n4js.runner.chrome/src/org/eclipse/n4js/runner/chrome/ChromeRunner.java

```diff
@@ -72,10 +72,10 @@ public Process run(RunConfiguration runConfig, IExecutor executor) {
 
 			final List<Path> paths = new ArrayList<>();
 
-			runOptions.addInitModules(runConfig.getInitModules());
-			if (runConfig.isUseCustomBootstrap()) {
```

Comment:
The ChromeRunner isn't used at the moment and contains invalid code anyway. I wanted to preserve the original code as reference to make it easier when updating the code at some point. I had to comment out some stuff to avoid compilation errors.

It's unfortunate but cause by the fact that we have out-dated, basically dead code in the repo we do not want to delete altogether for various reasons.